libraries/windows_helper.rb

```diff
@@ -86,17 +84,6 @@ def cached_file(source, checksum = nil, windows_path = true)
       end
     end
 
-    # Expands the environment variables
-    def expand_env_vars(path)
```

Comment:
We probably want this still. I'm sure it's used somewhere.

Reply:
Ref: https://github.com/chef/chef/commit/561c06d451427d444217f6d901a16f30144a6903#diff-6f1a02d1cc16201c682598cb413ebd42L52